common/supplemental/units.xml

```diff
@@ -90,7 +92,22 @@ For terms of use, see http://www.unicode.org/copyright.html
 		<convertUnit source='cup-metric' target='cubic-meter' factor='0.00025'/>
 		<convertUnit source='liter' target='cubic-meter' factor='0.001'/>
 		<convertUnit source='pint-metric' target='cubic-meter' factor='0.0005'/>
-		<convertUnit source='mile-per-gallon' target='liter-per-kilometer' factor='235.215' reciprocal='TRUE'/>
-		<convertUnit source='centimeter' target='meter' factor='1/100'/>
+		<convertUnit source='centimeter' target='meter' factor='1/100'/><convertUnit source='century' target='year' factor='100'/>
```

Comment:
May I suggest also adding the following since they exist directly in the CLDR locals?
```
    <convertUnit source='millimeter' target='meter' factor='1/1000'/>
    <convertUnit source='kilometer' target='meter' factor='1000'/>
```